gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/processor/transaction/TransactionResponseProcessorConfiguration.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.handlers.api.processor.transaction;
+
+import static io.gravitee.gateway.reactor.processor.transaction.TransactionHeader.DEFAULT_REQUEST_ID_HEADER;
+import static io.gravitee.gateway.reactor.processor.transaction.TransactionHeader.DEFAULT_TRANSACTION_ID_HEADER;
+
+public class TransactionResponseProcessorConfiguration {
+
+    public final String transactionHeader;
+
+    public final HeaderOverrideMode transactionHeaderHeaderOverrideMode;
+
+    public final String requestHeader;
+
+    public final HeaderOverrideMode requestHeaderHeaderOverrideMode;
+
+    public TransactionResponseProcessorConfiguration(io.gravitee.node.api.configuration.Configuration configuration) {
```

Comment:
Do you need that fully qualified name here ?
```suggestion
    public TransactionResponseProcessorConfiguration(Configuration configuration) {
```

Reply:
Updated